python/python/glide/async_commands/core.py

```diff
@@ -4288,3 +4360,43 @@ async def object_refcount(self, key: str) -> Optional[int]:
             Optional[int],
             await self._execute_command(RequestType.ObjectRefCount, [key]),
         )
+
+    async def getex(
+        self,
+        key: str,
+        expiry: Optional[ExpiryGetEx] = None,
+    ) -> Optional[str]:
+        """
+        Get the value of `key` and optionally set its expiration. `GETEX` is similar to `GET`, but is a write command with `ExpiryGetEx` options.
```

Comment:
```suggestion
        Get the value of `key` and optionally set its expiration. `GETEX` is similar to `GET`, but it is a write command.
```
or
```suggestion
        Get the value of `key` and optionally set its expiration. `GETEX` is similar to `GET`.
```
It is always a write command, even when no options given - will fail on a replica.